src/dosbox.cpp

```diff
@@ -834,6 +834,23 @@ void DOSBOX_Init() {
 	        "Provider for the OPL emulation. 'compat' provides better quality,\n"
 	        "'nuked' is the default and most accurate (but the most CPU-intensive).");
 
+	Pstring = secprop->Add_string("sb_filter", when_idle, "auto");
+	Pstring->Set_help(
+	        "Type of filter to emulate for the Sound Blaster digital sound output:\n"
+	        "  auto:    Use the appropriate filter determined by 'sbtype' (default).\n"
+	        "  sb1, sb2, sbpro1, sbpro2, sb16:\n"
+	        "           Use the filter of this Sound Blaster model.\n"
+	        "  none:    Don't filter the output.\n"
+	        "A second 'always_on' parameter can be provided to disallow programs from turning the filter off.\n"
+	        "(Example: sbpro1 always_on)");
+
+	Pstring = secprop->Add_string("opl_filter", when_idle, "auto");
+	Pstring->Set_help("Type of filter to emulate for the Sound Blaster OPL output:\n"
+	                  "  auto:    Use the appropriate filter determined by 'sbtype' (default).\n"
+	                  "  sb1, sb2, sbpro1, sbpro2, sb16:\n"
+	                  "           Use the filter of this Sound Blaster model.\n"
+	                  "  none:    Don't filter the output.");
+
```

Comment:
There will be 9 filters in total for the different sound cards according to the [mixer diagram here](https://github.com/dosbox-staging/dosbox-staging/issues/1662#issuecomment-1125628958). As far as I can tell, only the SB PCM filters are toggleable by software, so no other filters will need the `always_on` special flag. So instead of repeating "Programs have no way to toggle the <xyz> filter, so it's always on" for all filters, I thought I'd only mention it for the SB PCM output.

But because the OPL output is closely related to the SB, maybe we can put this sentence there, but omit it for the other filters? Do you think that's a good solution?

Reply:
> only the SB PCM filters are toggleable by software, so no other filters will need the `always_on` special flag. So instead of repeating "Programs have no way to toggle the filter, so it's always on" for all filters, I thought I'd only mention it for the SB PCM output.
> 
> But because the OPL output is closely related to the SB, maybe we can put this sentence there, but omit it for the other filters? Do you think that's a good solution?

Thanks for explaining that; the approach sounds good - or really any other way you'd like to describe it.